Terasology/MasterOfOreon
# Guard's behavior

Adds the `Guard` task performance logic.

### How to test : 
- Use the `Upgrade Tool` on any block of the building to trigger a screen and use the `Guard this building` button to add a `Guard` task to the Holding.
- When an Oreon is assigned the task it should walk around the building while performing the task.

#### Outstanding
- [x] Needs some import statement optimisations and style fixes.
